CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ The changelog for `IGListKit`. Also see the [releases](https://github.com/instag
 
 - Remove `IGListStackedSectionController`. [Hanton Yang](https://github.com/hanton) [(#1355)](https://github.com/Instagram/IGListKit/pull/1355)
 
+- Add Swift refinements for `dequeueReusableCellOfClass` methods. [Koen Punt](https://github.com/koenpunt) [(#1388)](https://github.com/Instagram/IGListKit/pull/1388)
```

Comment:
Can you add this under 4.1.0 ? If it's not created feel free to create one entry, thanks!